README.md

```diff
@@ -1,2 +1,17 @@
 # hw01
 Repository for homework 1
+
+Repository for first CFSS homework assignment
+
+hello! my name is Jenny Lu and I'm a PhD student in the **Psychology** Department.
+
+*A few facts about myself
+
+1. I'm originally from Boston
+2. I love to cook using different cooking techniques like sous vide
+3. I like to do processes that involve fermentation such as making yogurt, bread, and beer
+
+I also run a workshop called ["Modalities of Lanugage"] (https://voices.uchicago.edu/modalitiesoflanguage/) where students and faculty come together to discuss interesting current research on sign language and gesture.
```

Comment:
Your link doesn't appear inline here; try taking away the quotes around your text inside of the bracket.